site/_scss/blocks/_select.scss

```diff
@@ -0,0 +1,36 @@
+select {
+  appearance: none;
+  // sass-lint:disable quotes
+  background-image: url("data:image/svg+xml,%3Csvg width='24' height='24' viewBox='0 0 24 24' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M7 10L12 15L17 10H7Z' fill='%235F6368'/%3E%3C/svg%3E%0A");
+  background-position: right 0.5em top 50%;
+  background-repeat: no-repeat;
+  background-size: 1.5em;
+  border: 1px solid var(--color-hairline);
+  border-radius: 1px;
+  box-sizing: border-box;
+  display: block;
+  font-size: 1em;
+  line-height: 1.5;
+  margin: 0;
+  max-width: 100%;
+  padding: 0.6em 3em 0.5em 0.8em;
+}
+
+select::-ms-expand {
+  display: none;
+}
+
+// This will remove the dotted outline inside of the select when it is focused
+// in Firefox.
+// The trick is to make the text transparent (which makes the dotted outline
+// transparent) and then "display" the text again using a 0 offset text-shadow.
+// Note that this forces the text to be black when it is focused in Firefox.
+// To change that color, change the value of the text-shadow.
+select:-moz-focusring {
+  color: transparent;
+  text-shadow: 0 0 0 #000;
+}
```

Comment:
I think you don't have to do this anymore. Firefox recently had a visual refresh which changes the default style of form elements. I've checked with Firefox on macOS and Windows and they don't show dotted outlines inside select box.

Reply:
Done!